pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -83,6 +83,11 @@ func (c *Controller) Create(b *v3.EtcdBackup) (runtime.Object, error) {
 		cluster.Spec.RancherKubernetesEngineConfig.Services.Etcd.BackupConfig == nil {
 		return b, fmt.Errorf("[etcd-backup] cluster doesn't have a backup config")
 	}
+	if cluster.Spec.RancherKubernetesEngineConfig != nil &&
+		cluster.Spec.RancherKubernetesEngineConfig.Services.Etcd.BackupConfig != nil &&
+		!*cluster.Spec.RancherKubernetesEngineConfig.Services.Etcd.BackupConfig.Enabled {
+		return b, fmt.Errorf("[etcd-backup] cluster backup is disabled")
```

Comment:
are we sure we want to return error here if it's just disabled?

Reply:
Yes. We disable the cluster action if backup is disabled. So, to get here, the user would have to create a backup object using kubectl for a cluster with backup disabled. I consider that an error since it's not in any normal workflow.

Reply:
got it, thanks for detailed explanation! could you just check if we need to add cluster name to the msg